akka-docs/src/test/scala/docs/config/ConfigDocSpec.scala

```diff
@@ -109,4 +109,19 @@ class ConfigDocSpec extends AnyWordSpec with Matchers {
     val system = ActorSystem(rootBehavior, "MySystem", conf)
     ActorTestKit.shutdown(system)
   }
+
+  def compileOnlyIsLicenseKeyValid(): Unit = {
+    //#check-is-key-valid
+    val system = ActorSystem(rootBehavior, "name")
+    val licenseKey = system.licenseKeyExpiry
+    licenseKey.isEmpty shouldBe false
+
+    import java.time.LocalDate
+    val lastMonth = LocalDate.now().minusMonths(1)
+    licenseKey.get.isBefore(lastMonth) shouldBe true
```

Comment:
Haven't we got this backwards?
Let's say that my key expires March 25.
When I run the test today Feb 26 I want to see that as a failure.

```
val nextMonth = LocalDate.now().plusMonths(1)
licenseKey.get.isAfter(nextMonth) shouldBe true
```

Reply:
Uhh, jea of course 🤦‍♂️